pkix.go

```diff
@@ -28,126 +28,145 @@ func CrlTag(prefix string) *depot.Tag {
 	return depot.CrlTag(prefix)
 }
 
-// GetNameFromCrtTag returns the host name from a certificate file tag.
+// GetNameFromCrtTag returns the host name from a certificate tag.
```

Comment:
It doesn't have to be a host name (e.g. "evergreen" is not a host name). Can we call it something else, like "name on the certificate" or something.